app/controllers/meals_controller.rb

```diff
@@ -11,9 +12,26 @@ def new
 	end
 
 	def create
- 		@fave_num = params[:meal][:fave_num]
- 		@meal_num = params[:meal][:meal_num]
- binding.pry
+		destroy_all_user_meals
+ 		@fave_num = params[:meal][:fave_num].to_i
+ 		@meal_num = params[:meal][:meal_num].to_i
+
+ 		@meals = Meal.make_meals(@meal_num, current_user)
+
  		redirect_to meal_plan_path
 	end
+
+	private
+	
+	def destroy_all_user_meals
+		Meal.where(user_id: current_user.id).destroy_all
+	end
+
+	def readable_ingredients(meals)
+		(meals.map do |meal| 
+					meal.ingredients.map do |ingredient|
+						ingredient.name
+					end
+				end).flatten.uniq.sort
```

Comment:
`end` at 35, 4 is not aligned with `meals.map do |meal| ` at 31, 3 or `(meals.map do |meal| ` at 31, 2<br>Tab detected.